### DISCUSSION TITLE
Parameter link and Blender

### DISCUSSION THREAD

**Discussion by AndreaMonzini** [gianni-rosato/svt-av1-psy#23] [Ideas]
Hello,  

just to notice that the "full parameters description" link is broken:

https://github.com/gianni-rosato/svt-av1-psy/blob/master/Docs/CommonQuestions.md#8-or-10-bit-encoding

beside that  i wrote to consider  svt-av1-psy for the Blender "Support higher bits per pixel formats": 

https://projects.blender.org/blender/blender/issues/118493#

thank you

**Comment by gianni-rosato** [comment]
Hi,

Thanks for pointing out the documentation error! I have resolved it with 0b50de1f5730d381eee5cdd08ab9172730b7fdd2 in the `testing` branch. It'll be merged to master very soon.

Can you elaborate a bit more about the Blender issues & what Blender is looking for, as well as why you've recommended SVT-AV1-PSY?

**Comment by AndreaMonzini** [comment]
> Can you elaborate a bit more about the Blender issues & what Blender is looking for, as well as why you've recommended SVT-AV1-PSY?

i partecipated in a chat discussion ( Blender VSE ) about new output 10 bit format for Blender:

https://blender.chat/channel/sequencer-module

so i wrote that  AV1 supports 10 bit and i was thinking that SVT-AV1-PSY could be a good option after my testing with Aviator :)

https://projects.blender.org/blender/blender/issues/118493#

**Comment by gianni-rosato** [comment]
Is Blender's FFmpeg compiled with `--enable-libsvtav1` by default already? And are you involved with the Blender project?

**Reply by AndreaMonzini** [reply]
i try to help Blender project with discussions and reporting issues using Blender as creative tool.
I compiled Blender in the past and i plan to study more Blender code but i am not a Blender developer ( no commits for now ).
I do not know how FFmpeg is compiled in Blender.

About VFX & Video here you can see the module owner and developers:

https://projects.blender.org/blender/blender/wiki/Module:-VFX-&-Video

and there are interesting discussion in the sequencer-module ( Blender VSE ) chat:

https://blender.chat/channel/sequencer-module

**Reply by gianni-rosato** [reply]
Gotcha, I'll have a look

**Comment by gianni-rosato** [comment]
@AndreaMonzini I moved this to Discussions so we can have a bigger conversation about Blender, and how we can potentially bring SVT-AV1-PSY to bigger projects outside of Aviator.

**Reply by AndreaMonzini** [reply]
i think that SVT-AV1-PSY would be a great contribution for Blender if the implementations is possible of course.
This is an interesting period for Blender Video Sequencer Editor because the development is very active recently so there are discussions also about new 10 bit video output for compressed codecs.
